source/docs/zero-to-robot/step-4/creating-test-drivetrain-program-cpp-java.rst

```diff
@@ -59,6 +59,33 @@ For C++ projects, there is one more step to set up IntelliSense.  Whenever we op
 .. image:: /docs/software/vscode-overview/images/importing-previous-project/cpp-configurations.png
     :alt: You must choose "Yes" to refresh the C++ configurations.
 
+Basic Drivetrain example
+------------------------
+
+First, here is what a simple code can look like for a Drivetrain with PWM controlled motors (such as SparkMax).
```

Comment:
Say that for Java/C++ this is the Getting Started example as configured above, and add link for where to get the python getting started example